### ISSUE TITLE
asdf-user package

### ISSUE THREAD

**Issue by thibaultlanglois** [tokenrove/imago#1]
Hi, The recent change of the use-package in imago.asd beaks with quicklisp.
The package asdf-user does not exist.
T.

**Comment by tokenrove** [comment]
Thanks for the report.

What version of ASDF are you using?

According to the ASDF manual, this is the recommended form:
  https://common-lisp.net/project/asdf/asdf/The-defsystem-form.html
but if there are people actively using ancient versions of ASDF, I could certainly switch it back to the old form.

**Comment by thibaultlanglois** [comment]
Thnks for answering,
You are perfectly right ! 
I am using SBCL 1.1.3 which comes with ASDF2. I guess I'll have to upgrade someday.
Maybe imago's system definition could be more flexible with respect to ASDF version.
Do you think 
#+ASDF2
(in-package :cl-user)
 #+ASDF3
(in-package :asdf-user)
 could solve the issue ? (maybe you are using some ASDF3 specific feature.)

**Comment by tokenrove** [comment]
Well, on one hand I would like to encourage you to upgrade; there will be more and more packages that depend on ASDF3 features, so it can't be avoided forever.

On the other hand, I don't want to break anyone's existing setups with modifications I'm making to imago now.  After all, the whole reason I'm maintaining it is that I also have a body of older code that depends on it.

If you decide that you don't want to upgrade, or if someone else chimes in here to complain about the same issue, then I'll revert that change to something compatible with ASDF2.  However, if you're willing to upgrade, I'd rather go forward with ASDF3.  So let me know.

(By the way, also let me know if there are specific features in imago you are particularly missing; I am planning to do some updates to its handling of various planar image formats in the near future and would be happy to include other modifications at the same time.)

**Comment by thibaultlanglois** [comment]
OK. I have upgraded to last version of SBCL and I now have ASDF3. Problem solved.
I have been using imago sporadically for quite a long time now. I apreciate the clarity of the code. Once I needed to write strings on an image and I made this : https://dl.dropboxusercontent.com/u/886471/imago-fonts.zip
If you find it moderately useful you can include it.

**Comment by tokenrove** [comment]
Cool; thanks.  I will check out the font code and consider it or maybe a related implementation (aside from the obvious of using something like zpb-ttf and vecto, it might be nice to provide convenience functions to parse the output of bitmap font creators like http://kvazars.com/littera/).

Feel free to open another issue if there's anything else.
